### ISSUE TITLE
Add an option to auto saveedit

### ISSUE THREAD

**Issue by logan001** [qgis/QGIS#53572]
### Feature description

As the title says. i really find it annoying to have to start edit/save edit every time i want to edit something. also with a software not so stable as qgis where crashes are very common i really find it annoying to lose half  an hour or hours of work cause i forgot to save the edits of all layers.

i use to work with geomedia at work a few years ago  and that software was such easy to use for edit and nothing to lose when everything was save the moment you stop editing.

also if working with databases in multi user environment i think its also nice to have this implemented instead of being ask to save after every little edit.

**Comment by agiudiceandrea** [comment]
@logan001, doesn't the autoSaver plugin https://plugins.qgis.org/plugins/autoSaver/ meet your needs?

**Comment by logan001** [comment]
nope. i think i didn't make it clear enough. 
i need to commit every change  directly to database without the current buffer that exists. the autosave plugin has only a timer.

**Comment by alexbruy** [comment]
Duplicates #14115.

**Comment by agiudiceandrea** [comment]
@alexbruy , it seems this issue is not exactly a duplicate of the one linked by you.

**Comment by logan001** [comment]
@alexbruy not really. i want that after every edit to be committed directly in database without any type of save (manual or timer). i guess this implies that the row affected (in case of editing an existing data) to be locked in database to prevent unwanted edits.

**Comment by logan001** [comment]
oops i close the wrong one
